### PR TITLE
Scripts/Scholomance: Lady Illucia Barov spell corrections

### PR DESCRIPTION
**Pull request by Addra**
Correct 2 spells that arrent wrong here

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp [hunk 1]
```diff
@@ -30,8 +30,8 @@ enum Spells
 {
     SPELL_CURSEOFAGONY          = 34794,
     SPELL_SHADOWSHOCK           = 34799,
-    SPELL_SILENCE               = 34803,
-    SPELL_FEAR                  = 34803
+    SPELL_SILENCE               = 15487,
```
Comment (Aokromes) [inline src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp]: original ones was:
 -#define SPELL_SILENCE           15487
 -#define SPELL_FEAR              6215

**Comment by offl** [comment]
> Wiki
Curse of Agony - Curses nearby enemies with agony, inflicting Shadow damage every 3 sec. for 15 sec. Only one curse per warlock can be active on any one target.
Dominate Mind - Takes control of a humanoid enemy up to level 25 for 10 sec., but increases the time between its attacks by 100%.
Fear - Strikes fear in an enemy, causing it to flee in terror for 4 sec. Only 1 target can be feared at a time.
Shadow Shock - Instantly lashes an enemy with dark magic, inflicting Shadow damage.
Silence - Silences nearby enemies, preventing them from casting spells for 5 sec.

```sql
ACID
('1050201','10502','0','0','100','3','5000','11000','12000','19000','11','12528','0','0','0','0','0','0','0','0','0','0','Lady Illucia Barov - Cast Silence'),
('1050202','10502','0','0','100','3','9000','15000','11000','16000','11','17289','4','0','0','0','0','0','0','0','0','0','Lady Illucia Barov - Cast Shadow Shock'),
('1050203','10502','0','0','100','3','14000','18000','30000','36000','11','18671','0','1','0','0','0','0','0','0','0','0','Lady Illucia Barov - Cast Curse of Agony'),
('1050204','10502','0','0','100','3','21000','32000','18000','24000','11','12542','4','1','0','0','0','0','0','0','0','0','Lady Illucia Barov - Cast Fear'),
('1050205','10502','0','0','100','3','17000','23000','21000','26000','11','14515','5','0','0','0','0','0','0','0','0','0','Lady Illucia Barov - Cast Dominate Mind'),
```
--

Only one low level Curse of Agony lasts 15 sec - 18266.
Only one low level Dominate Mind lasts 10 sec - 7645.
With Fear everything is right, Fear from ACID lasts 4 sec (12542).
And seems like Shadow Shock from ACID is correct (17289).
18327 looks like correct Silence spell.

So this is more correct spell list:
Curse of Agony 18266
Dominate Mind 7645
Fear 12542
Shadow Shock 17289
Silence 18327

File: src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp (hunk 1 above)
Comment (joschiwald) [inline src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp]: this needs to be confirmed with sniffes, even curse of agony and shadow shock have wrong spellids

Comment (Shauren) [inline src/server/scripts/EasternKingdoms/Scholomance/boss_illucia_barov.cpp]: Except this might be hard - its old scholomance after all

**Comment by Addra** [comment]
The Boss calls Adds from a Raid/Dungeon from the Outlands. That fix i made correct this and the Fear was wrong and  tried to correct them

Did i something wrong?

**Comment by Aokromes** [comment]
ServerToClient: SMSG_SPELL_START (0xCE43) Length: 35 ConnIdx: 0 Time: 2011 
Caster GUID: Full:  Type: Creature Entry: 10502 Low: 49
Caster Unit GUID: Full:  Type: Creature Entry: 10502 Low: 49
Cast Count: 0
Spell ID: 17234 (17234)
Cast Flags: 2 (HasTrajectory)
Time: 0
Target Flags: 2 (Unit)
Target GUID: Full: Type: Player Low:  Name: 

TC>17234 - Shadow Shock, rank 1 enUS

ServerToClient: SMSG_SPELL_START (0xCE43) Length: 43 ConnIdx: 0 Time:2011
Caster GUID: Full: Type: Creature Entry: 10502 Low: 49
Caster Unit GUID: Full: Type: Creature Entry: 10502 Low: 49
Cast Count: 0
Spell ID: 12542 (12542)
Cast Flags: 67108866 (HasTrajectory, Immunity)
Time: 1500
Target Flags: 2 (Unit)
Target GUID: Full: Type: Player Low: Name:
CastSchoolImmunities: 0
CastImmunities: 67109376

TC> 12542 - Fear, rank 1 enUS

ServerToClient: SMSG_SPELL_START (0xCE43) Length: 43 ConnIdx: 0 Time: 2011
Caster GUID: Full: Type: Creature Entry: 10502 Low: 49
Caster Unit GUID: Full: Type: Creature Entry: 10502 Low: 49
Cast Count: 0
Spell ID: 7645 (7645)
Cast Flags: 67108866 (HasTrajectory, Immunity)
Time: 2000
Target Flags: 2 (Unit)
Target GUID: Full: Type: Player Low: Name:
CastSchoolImmunities: 0
CastImmunities: 67109376


TC>7645 - Dominate Mind, rank 1 enUS

**Comment by Aokromes** [comment]
Theorical:
Dominate Mind 7645
Fear 12542
Shadow Shock 17289
Curse of Agony 18266
Silence 18327

from 4.2.2 sniffs:
TC>7645 - Dominate Mind, rank 1 enUS
TC>12528 - Silence, rank 1 enUS
TC>12542 - Fear, rank 1 enUS
TC>17234 - Shadow Shock, rank 1 enUS
TC>18671 - Curse of Agony, rank 1 enUS

**Comment by Aokromes** [comment]
Since i cannot edit this. closing, i will push directly.

**Pull request closed without merging**